src/FunctionApps/python/GenerateCSVs/elr.py

```diff
@@ -1,8 +1,39 @@
 from typing import List, Any
-from GenerateCSVs.patient import PATIENT_COLUMNS
+from GenerateCSVs.patient import PATIENT_COLUMNS, parse_patient_resource
 
-ELR_COLUMNS = PATIENT_COLUMNS + []
+
+ELR_COLUMNS = PATIENT_COLUMNS + ["loinc_code", "result", "effective_date"]
```

Comment:
To align with the convention of matching FHIR variable names and using camel case could we change this to ["loincCode", "result", "effectiveDateTime"]?

Reply:
Yup sure thing!